pkg/ebpf/c/tracee.bpf.c

```diff
@@ -4219,6 +4221,36 @@ int uprobe_seq_ops_trigger(struct pt_regs *ctx)
     return 0;
 }
 
+SEC("uprobe/trigger_mem_dump_event")
+int uprobe_mem_dump_trigger(struct pt_regs *ctx)
+{
+    u64 address = 0;
+    u32 trigger_pid = bpf_get_current_pid_tgid() >> 32;
+
+#if defined(bpf_target_x86)
+    address = ctx->bx; // 1nd arg
+    char arch[] = "x86_64";
```

Comment:
Why is arch required? It is something that doesn't change on a given machine and you can get it in other ways

Reply:
how?

Reply:
One option - use uname. Many others exist.